Gemfile

```diff
@@ -75,3 +75,9 @@ gem 'will_paginate'
 gem 'devise', '~> 4.9'
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍